src/file.ts

```diff
@@ -2152,9 +2152,10 @@ class File extends ServiceObject<File> {
    *
    * @see [Policy Document Reference]{@link https://cloud.google.com/storage/docs/xml-api/post-object#policydocument}
    *
-   * @deprecated `getSignedPolicy()` is deprecated in favor of `getSignedPolicyV2()`
-   *     and `getSignedPolicyV4()`. Currently, this method is an alias to
-   *     `getSignedPolicyV2()`, and will be removed in a future major release.
+   * @deprecated `getSignedPolicy()` is deprecated in favor of
```

Comment:
`getSignedPolicy()` is alias to `generateSignedPostPolicyV2` and is slated for removal in the future, as discussed. @frankyn